docker/bin/start-worker-reload

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+WATCHED_FILES=$(python -c "print(' '.join([f'/src/{item}' for item in '$DEV_SOURCE_PATH'.split(':')]))")
+exec find $WATCHED_FILES \( -path /src/galaxy_ng/.venv -o -path /src/galaxy_ng/build -o -path /src/galaxy_ng/.eggs \) -prune -o -name '*.py' -o -name '*.env' | entr -n -r pulpcore-worker
```

Comment:
why the exec here?

Reply:
a successful exec never returns and that is how we make this to be the docker container command that never ends, otherwise it runs and stops the container soon.